LockboxXCUITests/LockboxXCUITests.swift

```diff
@@ -140,17 +142,21 @@ class LockboxXCUITests: BaseTestCase {
 
         XCTAssertTrue(app.navigationBars["Firefox Lockbox"].exists)
         XCTAssertTrue(app.navigationBars.buttons["Settings"].exists)
-        snapshot("02EntryList" + CONTENT_SIZE)
 
         sleep(5)
+        var i = 0
+        repeat {
+            i = i+1
+        } while (app.staticTexts["Confirm your account."].exists == true && i < 8)
```

Comment:
I was not happy with that solution either but need a workaround for issue #446. If that ```Confirm your account screen``` is shown, we go to verify the account so that we can log in after the account is verified. The problem is that sometimes that screen is shown for a while when the account does not need to be verified.
`waitForNoExistence`does not work, since it will fail in the case where we actually need to verify the account and that screen is not going to disappear until that's done. 
So we need to wait and check if the ```Confirm your account screen``` disappears after a while or not to continue with the test according to the real status of the account.

I will work on this and try to find a better solution.